editor/Dockerfile

```diff
@@ -0,0 +1,23 @@
+###########################
+#         Builder         #
+###########################
+
+FROM hub AS builder
+
+ARG version="2020.1.4f1"
+ARG changeset="fa717bb873ec"
+ARG module="android"
+
+# Install
+RUN unity-hub install --version $version --changeset $changeset --module $module
```

Comment:
I think '--cm' needs to be added here - this automatically installs the child modules (e.g. for android openjdk,sdk,ndk)

Reply:
Currently testing if that works for the `install` command, as it's only documented for the `install-modules` command.

Reply:
works